src/services/aggregatedLoggingService.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+angular.module("openshiftCommonServices")
+  .factory("AggregatedLoggingService", function($q, $cacheFactory, Logger, $interval, APIService, DataService) {
+
+    var currentUser = null;
+    var cachedUserPermissions = $cacheFactory('operationsUsersCache', {
```

Comment:
This seems like overkill since we're only ever checking one permission. It's really just one boolean value and a promise that we need to track.